src/config/index.ts

```diff
@@ -1,9 +1,17 @@
 export const ConfigIndex = {
-    database: {
-      host: process.env.DB_HOST,
-      user: process.env.DB_USER,
-      port:5432,
-      password: process.env.DB_PASS,
-      database: process.env.DB_NAME,
-    },
-  };
\ No newline at end of file
+  database: {
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    port:5432,
+    password: process.env.DB_PASS,
+    database: process.env.DB_NAME,
+  },
+};
+
+export  const typeMappings = {
```

Comment:
Can't we create a separate table for this?As with time,the number of languages could be more and we have to modify this typeMapping variable too.It would be a very lengthy variable.

Reply:
it is just a datatype like const values.  we want in object format. because when we are creating the default function it is easy. One more thing If we want to add a new language we need to do some research then we can add it.